pkg/apis/core/types.go

```diff
@@ -2380,7 +2383,13 @@ type PodSpec struct {
 	// before the system actively tries to terminate the pod; value must be positive integer
 	// +optional
 	ActiveDeadlineSeconds *int64
-	// Required: Set DNS policy.
+	// Set DNS policy for containers within the pod.
+	// One of 'ClusterFirstWithHostNet', 'ClusterFirst', 'Default' or `Custom`.
```

Comment:
Set DNS policy for the pod.
Defaults to "ClusterFirst".
Valid values are 'ClusterFirstWithHostNet', 'ClusterFirst', 'Default' or `Custom`.
DNS parameters given in DNSParams will be appended to the policy selected with DNSPolicy.

Reply:
Done.